.github/workflows/publish.yaml

```diff
@@ -59,5 +61,14 @@ jobs:
           install_expo: true
           expo_token: ${{ secrets.EXPO_TOKEN }}
 
+      - name: Replace IOS USER value in eas.json
+        run: sed -i 's/$IOS_USER_ID/${{ secrets.IOS_USER_ID }}/g' eas.json
```

Comment:
there is likely a better way to do this? let me know and I can change, I just went with the path of least resistance :D

Reply:
What we've done in the past is to add a placeholder into the `app.json`:

https://github.com/stevekuznetsov/avalanche-forecast/blob/a7ead5c25b18b2017371657fd7ece8a83744cda2/app.json#L28

Then overwrite it at load-time from the environment using TypeScript:
https://github.com/stevekuznetsov/avalanche-forecast/blob/a7ead5c25b18b2017371657fd7ece8a83744cda2/app.config.ts#L11

Let's check if we can also do this with `eas.json` - I would assume so!

Reply:
ok cool will test this today! I like this much better :)

Reply:
@stevekuznetsov something I realized, the eas.json is simply used for the eas submission, has nothing to do with the app, so loading values from env during app build doesnt help us here. Eas need to be populated in the github action so it can then use that info when it goes to submit the app. 

I am torn with just hardcoding those values in eas, what are your thoughts on that? are those values actually secret? I cant seem to find any info one way or another.

Reply:
ok hilarious, not sure if this is more expo magic but those values do not even have to be populated at all it seems - I created a build that had placeholders for them and the wrong apple password and the version of the app still submitted. so weird!